swift_scality_backend/diskfile.py

```diff
@@ -159,13 +156,13 @@ def zero_copy_send(self, wsockfd):
             if resp.status != httplib.OK:
                 raise SproxydHTTPException(
                     'Unexpected response code: %s' % resp.status,
-                    ipaddr=ipaddr, port=port, path=self.safe_path,
+                    url=sproxyd.geturl() + self._name,
```

Comment:
The `sproxyd` instance is a "Sproxyd client" instance, it has no idea of "objects". `geturl()` only returns an endpoint to a sproxyd connector. So it makes sense that the object named has to be appended.